speculos/api/events.py

```diff
@@ -27,6 +33,14 @@ def remove_client(self, client):
 
     def broadcast(self, event):
         self.logger.debug(f"events: broadcasting {event} to ({len(self.clients)}) client(s)")
+        if self.screen_content:
+            # Reset screen content if new event is not below
+            # the last text line of current screen.
```

Comment:
Adding in the comment that Y start from the top of the screen would make the following comparison unambiguous.

Reply:
Done.